adoptopenjdk-11/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM alpine:3.10 as maven-download
+
+RUN apk add --update curl \
+    && rm -rf /var/cache/apk/*
+
+ENV MAVEN_VERSION=3.6.3 \
+    SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
+ENV BASE_URL=https://archive.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/binaries
+
+RUN mkdir -p /usr/share/maven/; \
+    curl -fsSL -o /tmp/apache-maven.tar.gz ${BASE_URL}/apache-maven-${MAVEN_VERSION}-bin.tar.gz; \
+    echo "${SHA}  /tmp/apache-maven.tar.gz" | sha512sum -c -; \
+    tar -xzf /tmp/apache-maven.tar.gz -C /usr/share/maven --strip-components=1
+
+COPY settings-docker.xml /usr/share/maven/ref/
+
+FROM adoptopenjdk/openjdk11:x86_64-alpine-jdk-11.0.5_10-slim
```

Comment:
better to use the less specific image so it doesn't need to be updated continuously